modules/api/sync/package/index.py

```diff
@@ -18,23 +18,35 @@ def lambda_handler(_event, _context):
     logging.basicConfig(level=logging.DEBUG, format='%(asctime)s - %(levelname)s - %(message)s')
     logging.debug('Connected to the database')
 
-    # data_provider.get_data()
-    # todo replace the existing file reader by the file getted by the previous function
+    urls = get_data()
 
-    synchronize_decrees(cursor)
-    synchronize_restrictions(cursor)
+    synchronize_decrees(cursor, urls['decrees'])
+    synchronize_restrictions(cursor, urls['restrictions'])
 
     cursor.close()
     connection.close()
 
-def synchronize_decrees(cursor) -> None:
+def get(url):
+    '''gets csv from url and parse the data '''
+    response = requests.get(url)
+
+    if response.status_code != 200:
+        print(f"La requête à l'url {url} a échoué avec le code d\'erreur: {response.status_code}.")
```

Comment:
```suggestion
        logger.error(f"La requête à l'url {url} a échoué avec le code d\'erreur: {response.status_code}.")
```

J'ai mis les autres logs en anglais, c'est mieux si c'est cohérent mais j'ai pas d'avis sur le fr ou en.